src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj

```diff
@@ -4,6 +4,10 @@
     <Description>Zipkin exporter for OpenTelemetry</Description>
     <PackageTags>$(PackageTags);Zipkin;distributed-tracing</PackageTags>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\OpenTelemetry\Internal\EnumerationHelper.cs" Link="Implementation\EnumerationHelper.cs" />
+    <Compile Include="..\OpenTelemetry\Internal\PooledList.cs" Link="Implementation\PooledList.cs" />
+  </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\OpenTelemetry\OpenTelemetry.csproj" />
```

Comment:
do we still need  `<PackageReference Include="System.Text.Json" Version="4.7.0" />` as we are using the custom `Write` method now?

Reply:
Ya the write is using `Utf8JsonWriter` which comes out of System.Text.Json.